src/test/java/io/connect/scylladb/integration/ScyllaDbSinkConnectorIT.java

```diff
@@ -43,11 +45,12 @@
 
   private static final Logger log = LoggerFactory.getLogger(ScyllaDbSinkConnectorIT.class);
 
-  static final String SCYLLA_DB_CONTACT_POINT = "172.18.0.3";
+  static final String SCYLLA_DB_CONTACT_POINT = "0.0.0.0";
```

Comment:
It would be great if this IP address was dynamically set based on real Docker address, similarly as it's done in scylla-cdc-java packet scylla-cdc-driver3.